static/js/typing_events.js

```diff
@@ -62,7 +62,7 @@ export function render_notifications_for_narrow() {
         $("#typing_notifications").html(
             render_typing_notifications({
                 users: users_typing,
-                several_users: num_of_users_typing >= MAX_USERS_TO_DISPLAY_NAME,
+                several_users: num_of_users_typing > MAX_USERS_TO_DISPLAY_NAME,
```

Comment:
This seems to be a mistake I made in dba21d2. I think it's worth mentioning @timabbott regarding this.

Reply:
Can you adjust the tests to verify this bug fix as well?

Reply:
I guess by this I mean one in the actual typing subsystem tests.

Reply:
We don't seem to have tests for this file. Will try to open a new PR adding a `typing_events.js` in frontend_tests with a copy of this example test and add a few new ones for other functions in `static/js/typing_events.js`.

Reply:
Added a `typing_events.js` test file with the test for this here itself as another commit.